### ISSUE TITLE
Button missing a text alternative: section expand/collapse

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5570]
Sites: MCO, Archivo, IJCCR

Page: Media object page, structured navigation: expand/collapse button next to each section

WCAG Compliance Level: A

See more details in the [SiteImprove example](https://my2.siteimprove.com/Inspector/1166400/A11Y/Page?pageId=37945420394&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r12/failed/NWV5TkN2VnM=;OTZiZjdkNDM=).

<img width="1209" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0c5QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e2b7e8433e03f63fc0c869665cbc6ca2e019985a/image.png" alt="image.png" />

**Comment by elynema** [comment]
This will be irrelevant once 7.7 released, but our compliance date is early February, so we should try to fix this before 2/8 in current sites if possible.

**Comment by elynema** [comment]
This looks good. SiteImprove browser plugin identifies this problem on a page in MCO, but not on the same page in mco-staging. https://mco-staging.dlib.indiana.edu/media_objects/2z10x593s
